src/main/java/io/neow3j/examples/contractinvoke/EscrowExampleCreateTrustAgreement.java

```diff
@@ -0,0 +1,169 @@
+package io.neow3j.examples.contractinvoke;
+
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+import io.neow3j.contract.GasToken;
+import io.neow3j.contract.SmartContract;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.core.response.InvocationResult;
+import io.neow3j.protocol.core.response.NeoSendRawTransaction;
+import io.neow3j.protocol.core.stackitem.StackItem;
+import io.neow3j.protocol.http.HttpService;
+import io.neow3j.transaction.AccountSigner;
+import io.neow3j.types.ContractParameter;
+import io.neow3j.types.Hash160;
+import io.neow3j.wallet.Account;
+
+public class EscrowExampleCreateTrustAgreement {
+        public static void main(String[] args) throws Throwable {
+                /*
+                 * This example will create a Trust agreement between a trustor, an arbiter and
+                 * a beneficiary.
+                 * Initially, you will run the `createAgreement` function to create a Trust
+                 * agreement where you will specifiy who the arbiter and beneficiary is. You
+                 * will automatically be assigned as the trustor.
+                 * 
+                 * Then you will run the `getAgreement` function to get the details of the
+                 * agreement you just created so you can see the agreement.
+                 * 
+                 * Next you will run the `executeAgreement` function provided in the
+                 * `EscrowExampleExecuteTrustAgreemnt` class file to execute the agreement as
+                 * the arbiter. This will transfer the funds to the beneficiary.
+                 * 
+                 * In between the last 2 steps, you can run the `getContractBalance` utility
+                 * function to get the total GAS balance of the Escrow to check if your code
+                 * worked.
+                 */
+
+                // First we will do some intial setup work for the Trust contract
+                // 1. Instantiate a connection to the locally running Neo node
+                Neow3j neow3j = Neow3j.build(new HttpService("http://localhost:50012"));
+
+                // 2. Create a script hash of your locally deployed Escrow contract
+                Hash160 scriptHash = new Hash160("0a47419ae8f3c19fea7f8f6100ee6813cab3b54e");
+
+                // 3. Initialize the GAS token running on your local Neo node
+                GasToken gas = new GasToken(neow3j);
+
+                // 4. Initialize the trustor account
+                Account TRUSTOR_WIF = Account.fromWIF("L2g8aqX5WZS3n2ZqEtwHnYyBTY5K9RzvVsFc9zFNvSCxKFsMDx86");
+
+                // Function calls
+                createAgreement(TRUSTOR_WIF, scriptHash, neow3j);
```

Comment:
```suggestion
                Account trustorAccount = Account.fromWIF("L2g8aqX5WZS3n2ZqEtwHnYyBTY5K9RzvVsFc9zFNvSCxKFsMDx86");

                // Function calls
                createAgreement(trustorAccount, scriptHash, neow3j);
```

Maybe use `ALICE` as the trustor account (we should provide that account in the examples). It will already hold funds when using our `neo3-privatenet-docker` setup for a local Neo N3 network, so it's simpler to run the example.

Reply:
I have used `Alice` as the arbiter in this example. The user will have to create 2 more additional accounts to run this example anyway so shall I commit your suggestion and leave it as is? 🤔

Reply:
Okay, but then you can leave the current trustor, but please still commit the suggestion (renaming).